### PR TITLE
Update readme, add .npmrc

### PR DESCRIPTION
**Pull request by ayreal**
Updates to include what I needed to run the project on my machine. I haven't ever worked with pnpm so these additions may not be general. Open to feedback!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,15 +3,24 @@
...
 
 1. Clone the project
-2. From project root, run:
+2. Install and use the correct node version with:
+
+```shell
+nvm install 16.4 && nvm use 16.4
+```
+
+3. Install [pnpm](https://pnpm.io/installation#using-homebrew) via Homebrew
```
Comment (Timmehs) [inline README.md:18]: The `pnpm` install scripts are designed to work around the node version prerequisite. Once `pnpm` is installed, it will handle node version fetching and switching based on the repository. I think this step could be condensed to just:
```suggestion
2. Install [pnpm](https://pnpm.io/installation) for your operating system. (MacOS will use either of the [POSIX Systems scripts](https://pnpm.io/installation#on-posix-systems).
```

**Review by Timmehs — COMMENTED** [review]
So appreciative you thought to update the documentation, thank you 🙏 . This is my first project using pnpm myself, and this has made me realize that it's got a bunch of NodeJS version handling seamlessly built into it, which is rad.

I personally don't use nvm (nor does @eddie-m-m), and I think pnpm has just magicked away having the worry about node version. I was able to `p install` after pulling your branch, and it added node 16 to my system (p is my `pnpm` alias, how lazy is that):
```
$ p i
Fetching Node.js 16.14.0 ...
Lockfile is up to date, resolution step is skipped
Already up to date
Done in 6.9s
```
^^ I was on node 18 globally. All of the node version management commands are under `p env`, e.g.:
```
$ p env list
  16.14.0
  20.9.0
```

The TLDR is that if we direct new devs to the installation scripts first, it streamlines the instructions and they don't need to worry about installing a particular node version. Would love to hear from @eddie-m-m if he has any input, since he's set up a few environments himself at this point.

File: README.md (hunk 1 above)
Comment (eddie-m-m) [inline README.md:18]: I think this can be shortened to just the first suggested sentence to keep it as generic as possible.

```suggestion
2. Install [pnpm](https://pnpm.io/installation) for your operating system.
```

**Review by eddie-m-m — COMMENTED** [review]
I think in general a person only needs to get pnpm on their system (using their preferred method of course), and adding more steps at the general level is probably unnecessary at this time.

I use mac and linux: on mac I've installed with `brew install pnpm` and on linux with `npm install -g pnpm`. pnpm took it from there : )

@Timmehs covered the pros of pnpm for the workflow and that has been my  experience.

**Review by eddie-m-m — APPROVED** [review]
LGTM

**Comment by ayreal** [comment]
Thank you so much for the great feedback, y'all!
